test/TesterInternal/StorageTests/SQLAdapter/temp/TestEnvironmentInformation.cs

```diff
@@ -0,0 +1,39 @@
+﻿namespace UnitTests.StorageTests.SQLAdapter.temp
+{
+    /// <summary>
+    /// Enumeration of the supported test environment configurations.
+    /// </summary>
+    public enum SupportedTestConfigurations
+    {
+        None                        = 0,
+        Development                 = 1,
+        Jenkins                     = 2,
+        VisualStudioTeamServices    = 3
+    }
```

Comment:
Regarding the environment enumeration, I would not go with that approach. I would instead allow for certain fallbacks, but not make it named-environment dependant. We already have such a (very rough) implementation of fallback with `StorageTestConstants`. I don't like that approach in particular, and it could be improved a lot, but still I would like that for people who have their environments set up correctly, that those tests run, regardless of a value he uses in the enumeration (because he might have SQL connection strings, but not the azure storage ones, and each fixture should pick up correctly regardless of the other, and not be bound to the same enum value).